changelog.txt

```diff
@@ -1,3 +1,14 @@
+---------------------------------------------------------------------------------------------------
+Version: 0.5
+Date: 2021-03-04
+
+  Changed:
+    - New updated graphics by snouz:
+    - New building visuals, HR and LR, with normalized shadows, separate wire connector
+    - New icons for buildings and technologies
+    - New mod icon
+    - Logistic warehouse technology now requires "logistic robotics"
```

Comment:
This is best summarized in the PR description (top comment). I write the changelog based on what PRs were merged for each release, rather than trying to update the changelog in every PR.